package/MDAnalysis/analysis/utils/autocorrelation.py

```diff
@@ -21,44 +21,120 @@
 # J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
 #
 
```

Comment:
How about moving it to a new module `lib/correlations.py` – I don't want to bloat `mdamath` even further and the functions here are still fairly specific though independent of the core MDA stuff: thus `lib` is a good place and a separate module is also appropriate.